README.md

```diff
@@ -23,31 +23,49 @@ with the json files in the configs folder of rbac-config.
 Contributing
 =============
 
-Add new roles
+Canned roles
 -------------
+
+Add new roles
+
 Follow existing examples to add roles including name, description, system flag, access with permissions.
 If you want the new role to be associated with platform default group (which defines the default permissions
 for principals in a tenant), you have to add the platform_default flag and set it as true.
-
 Set the version to 2 for the new role in order to trigger the seeding in the rbac service.
-
-Format of permissions
-~~~~~~~~~~~~~~~~~~~~~
-The permission should have the format of <application>:<resource>:<action>. There are no restrictions on the
+If you have a new permission added, don't forget to add it to the permissions folder too. See "Add new permissions"
+section below for more details.
+~~~~~~~~~~~~~~~~~~~~~~
+Format of permissions:
```

Comment:
I think this is causing the formatting to be off. It's displaying odd for me.
<img width="1016" alt="Screen Shot 2020-06-09 at 11 13 17 AM" src="https://user-images.githubusercontent.com/1641557/84165916-3dd2a580-aa42-11ea-80d4-083f861a7bd3.png">

Reply:
Oh, I deliberately make it look like this. First level is Add new/Update/Delete roles. Then the "Format of permissions" is a kind of high light. 

Are you referring to this high light? Do you want me to make it same level as the "Add new/Update/Delete roles"?

Reply:
Updated, I put them to the same level with the others 😉